packages/labs/context/src/lib/value-notifier.ts

```diff
@@ -57,8 +57,11 @@ export class ValueNotifier<T> {
           this.callbacks.delete(callback);
         });
       }
+      const dispose = this.callbacks.get(callback)!;
```

Comment:
maybe while you're here: `callbacks` isn't a great name. I usually like naming a map for the values it contains, not the key. I think this would be better named as `disposers`.